atlasdb-jepsen-tests/src/jepsen/atlasdb/lock.clj

```diff
@@ -91,18 +92,23 @@
           ;; 15% chance of a lock
           0.85 {:type :invoke   :f :lock      :value random-lock-name}
           ;; 15% chance of an unlock
-               {:type :invoke   :f :unlock    :value random-lock-name})))))
+          {:type :invoke   :f :unlock    :value random-lock-name})))))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Defining the Jepsen test
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 (defn lock-test
-  []
+  [nem]
   (assoc tests/noop-test
     :client (create-client nil nil nil)
+    :nemesis nem
     :generator (->> generator
-                    (gen/stagger 0.1)
-                    (gen/clients)
-                    (gen/time-limit 30))
+                 (gen/stagger 0.05)
+                 (gen/nemesis
+                   (gen/seq (cycle [(gen/sleep 5)
+                                    {:type :info, :f :start}
+                                    (gen/sleep 15)
```

Comment:
Is 15 seconds appropriate for all the nemeses? How long does the server take to start up?